src/libraries/System.Linq/src/System/Linq/OrderedEnumerable.cs

```diff
@@ -7,618 +7,686 @@
 
 namespace System.Linq
 {
-    internal abstract partial class OrderedEnumerable<TElement> : IOrderedEnumerable<TElement>
+    public static partial class Enumerable
```

Comment:
Guessing we nest the other iterator types?

Reply:
Yup. These was the odd-man out, and Iterator itself is nested, which makes it more awkward for types not also nested.